embulk-core/src/test/java/org/embulk/spi/time/TestTimeZoneIds.java

```diff
@@ -1,18 +0,0 @@
-package org.embulk.spi.time;
```

Comment:
Tested in https://github.com/embulk/embulk-util-timestamp/blob/v0.2.1/src/test/java/org/embulk/util/timestamp/TestLegacyDateTimeZones.java